package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@blackbaud/auth-client",
-  "version": "2.38.0",
+  "version": "2.39.0",
```

Comment:
I would argue this is a bug fix, so maybe `2.38.1`?

Reply:
Fair enough. I've updated this. I'll update the PR's title appropriately.